### PR TITLE
CLI-25 - Fix VPC Flow logs support

### PR DESCRIPTION
**Pull request by silvexis**
CLI-25 - Fix Bugs
CLI-1 - Reactor info function

### CODE REVIEW

File: cloudzerocli/features/aws/connect.py [hunk 1]
```diff
@@ -34,10 +37,9 @@ def functions(arguments, reactor_name, settings, aws):
     else:
         check_aws_account(aws)
         account_name = arguments.get('--account-name') or aws.account_name()
-        account_name = account_name.lower()
         if not account_name:
-            print('ERROR: Could not automatically determine the name of your account, please specify a\n'
-                  '       --account-name <name> via the command line and try again')
+            print('ERROR: Could not automatically determine the name of your account, please use\n'
+                  '       --account-name <name> to specify a account name and try again')
```
Comment (Tankanow) [inline cloudzerocli/features/aws/connect.py]: ... to specify **_an_** account name ...

File: cloudzerocli/features/aws/helpers.py [hunk 2]
```diff
@@ -564,14 +560,14 @@ def __enable_vpc_flow_logs(self):
...
+                    retentionInDays=1
                 )
                 if response['Unsuccessful']:
                     print('     - Could not create VPC Flow Logs for {}'.format(vpc.id))
                 else:
-                    print('     - Enabled VPC Flow log /aws/flow-logs/{vpcid})'.format(vpcid=vpc.id))
-            else:
-                print('     - VPC flow logs for {} already enabled ({})'.format(vpc.id, flow_logs[vpc.id]))
+                    print('     - Enabled VPC FlowLog {})'.format(log_group_name))
+            except botocore.exceptions.ClientError:
```
Comment (Tankanow) [inline cloudzerocli/features/aws/helpers.py]: Is this the only reason boto would throw a `ClientError`?

File: cloudzerocli/features/reactor/info.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+def info_functions(reactor_name, settings):
+    """Parse arguments to call appropriate `cz reactor info` function.
+
+    Args:
+        reactor_name (str):  Name of the reactor to view details on
+        settings (Settings): in-memory representation of reactor settings;
+                             changing this will potentially change files in ~/.cloudzero/
+    Ret:
+        completed (bool):    whether or not the process completed successfully
```
Comment (Tankanow) [inline cloudzerocli/features/reactor/info.py]: The code below does not support the documented return type.

File: cloudzerocli/features/aws/connect.py [hunk 4]
```diff
@@ -34,10 +37,9 @@ def functions(arguments, reactor_name, settings, aws):
     else:
         check_aws_account(aws)
         account_name = arguments.get('--account-name') or aws.account_name()
-        account_name = account_name.lower()
         if not account_name:
-            print('ERROR: Could not automatically determine the name of your account, please specify a\n'
-                  '       --account-name <name> via the command line and try again')
+            print('ERROR: Could not automatically determine the name of your account, please use\n'
+                  '       --account-name <name> to specify an account name and try again')
```
Comment (tiyberius) [inline cloudzerocli/features/aws/connect.py]: 👍

File: cloudzerocli/features/aws/helpers.py [hunk 5]
```diff
@@ -160,21 +161,14 @@ def setup(self, reactor_account_id, reactor_id, account_name):
...
     def verify(self, reactor_details, endpoint, credentials, settings):
         self.reactor_account_id = reactor_details['account_id']
         self.reactor_name = reactor_details['name']
-
-        # if we have an AWS connection, attempt to load cloudtrail details directly
-        # todo why?
-        if self.aws_session:
-            self.__load_cloudtrail_details()
-            self.aws_region_name = self.aws_session.region_name
-
```
Comment (tiyberius) [inline cloudzerocli/features/aws/helpers.py]: because if we have to ask "why?" it probably shouldn't be here

**Pull request merged**